tests/foreman/api/test_repository.py

```diff
@@ -1317,6 +1326,32 @@ def test_positive_sync_rh(self):
         )
         entities.Repository(id=repo_id).sync()
 
+    @tier2
+    @skip_if_not_set('fake_manifest')
```

Comment:
This decorator can be dropped, fake_manifest is needed only for manifest cloning / upload. Here, you only sync public repo.